portality/api/v2/data_objects/common_journal_application.py

```diff
@@ -13,4 +13,7 @@ class OutgoingCommonJournalApplication(SeamlessMixin, swagger.SwaggerSupport):
     @classmethod
     def from_model(cls, journal_or_app):
         d = deepcopy(journal_or_app.data)
+        # Prevent the field from appearing in the outgoing API models
+        if d["bibjson"]["oa_start"]:
+            del d["bibjson"]["oa_start"]
```

Comment:
This change is at odds with the change above which makes this field required for incoming articles.  It looks like this PR mixes some phase 1 and some phase 2 requirements.

Since we're going straight to phase 2, this bit of code can be removed, I think.

Reply:
This PR is already for phase 1 and phase 2 - I edited the first comment. Outgoing API is phase 3 though, that's why it should stay for now.